pontoon/base/static/css/users.css

```diff
@@ -85,13 +85,7 @@ sup a:visited {
 .contributor p.name {
   display: inline-block;
   font-size: 1.5em;
-  height: 1.5em;
-}
-
-.contributor p.email {
-  left: 67px;
-  position: absolute;
-  top: 47px;
+  line-height:44px;
```

Comment:
"Names" are now longer, because we often present email addresses instead. We need smaller font size. Maybe 16 px like in the sidebar in the translate view. Also, the actual height should be 48px (also in the translate view), like the image. Currently text is not vertially center-aligned to the image.

It would be nice to unify the HTML code we use in all these user presentation, but we can also do this in a separate PR.